OpenRA.Game/Game.cs

```diff
@@ -279,6 +269,25 @@ internal static void Initialize(Arguments args)
 				}
 			}
 
+			GeoIP.Initialize();
+
+			try
+			{
+				if (Game.Settings.Server.DiscoverNatDevices == false)
+					Game.Settings.Server.AllowPortForward = false;
+				else
+				{
+					UPnP.DiscoverNatDevices(Settings.Server.NatDiscoveryTimeout).Wait();
```

Comment:
Unfortunately this is still before the first load screen draw (which happens after ModData is created).
This whole block will either need to move later still, or you could keep it asynchronous, like the old code, to avoid stalling the init.